playbook/workshops/dodge/add_enemy_sprite.md

```diff
@@ -1,5 +1,8 @@
 # Add Enemy Sprite
 
+_Questions, comments and critiques are welcome! Send your feedback
```

Comment:
Do you think they will remember what they wanted to submit feedback on by the end of the workshop?